locales/base/translation.json

```diff
@@ -2484,17 +2484,19 @@
       "details": {
         "earn": {
           "titleGasSubsidy": "Maximize Earnings without Gas Fees*",
-          "title": "Maximize Earnings",
-          "subtitle": "Earn on your stablecoins and get rewards when you supply a pool.",
+          "title": "Explore and Compare Pools",
+          "subtitle": "Compare earning potential and risks. Swap, buy, or transfer tokens to join the best pool for you.",
```

Comment:
Related to the gas subsidy question, this change would mean that for older versions, with gas subsidy on, we'll still show the `titleGasSubsidy` as title, with this new subtile. Is that intended?

Reply:
Checked with product and they're ok showing the gas subsidy title on old app versions.